pyserverforindy/grpc_server/blob_storage_service.py

```diff
@@ -1,7 +1,9 @@
 import os
 import sys
 
-from indy import blob_storage as indy_blob_storage
```

Comment:
no major changes in anoncres_service. Just add IndyError handler for future changes